### DISCUSSION TITLE
remap o to open has a delay?

### DISCUSSION THREAD

**Discussion by biowaffeln** [nvim-neo-tree/neo-tree.nvim#1204] [Q&A]
In my config, I tried to remap "o" to "open", just like in netrw, like so:
```lua
mappings = {
	["o"] = { "open" },
}
```

However, everytime I press o there's a slight delay. I assume that's because o is a prefix for the order command. So how would I remap o to open without the delay? I tried setting all the order commands to a different key, like this:

```lua
mappings = {
	["o"] = { "open", nowait = true },
	["ö"] = { "show_help", nowait = false, config = { title = "Order by", prefix_key = "ö" } },
	["öc"] = { "order_by_created", nowait = false },
	["öd"] = { "order_by_diagnostics", nowait = false },
	["öm"] = { "order_by_modified", nowait = false },
	["ön"] = { "order_by_name", nowait = false },
	["ös"] = { "order_by_size", nowait = false },
	["öt"] = { "order_by_type", nowait = false },
},
```

But that didn't get rid of the delay on the o

**Comment by cdaein** [comment]
this worked for me:
https://github.com/nvim-neo-tree/neo-tree.nvim/issues/1128
